website/blog/2023-04-18-aluminium.md

```diff
@@ -0,0 +1,391 @@
+---
+author: Katarzyna Marek
+title: Metals v0.11.12 - Aluminium
+authorURL: https://github.com/kasiaMarek
+authorImageURL: https://github.com/kasiaMarek.png
+---
+
+We're happy to announce the release of Metals v0.11.12, which brings mostly stability fixes and some improvements. It also starts an effort towards improving identification of issues for users.
```

Comment:
> It also starts an effort towards improving identification of issues for users.

It sounds a bit unclear for me (because "improving problem identifications" sounds same as the "improving the stability").
What do you think make it more concrete like "This release makes it easier for users to report the bug report to metals"?

Reply:
I think it's a bit more than just reporting bugs. So in this release we only save more context information for an error state, but in general I think we're planning to improve how errors are handled: what diagnostics are collected, how errors are presented to the user, what actions are proposed  and so on... For me that's more about what we do when an error happens, where improving stability is more about avoiding errors.

Reply:
That makes sense :+1: and chris's rewording made it nicer, thank you for answering!